### PR TITLE
Add tests for simple graph, with multiple inputs

### PR DESCRIPTION
**Pull request by stangirala**
## Context

Add a couple of tests for a simple graph function where the function takes in multiple inputs. This should work given how we setup the args but we didn't have a test for it. Also add test, for sanity, with json encoding.

## What

Add tests for graph behaviours. This will run as a workflow action.

## Testing
N/A

## Contribution Checklist

- [ ] If the python-sdk was changed, please run `make fmt` in `python-sdk/`.
- [ ] If the server was changed, please run `make fmt` in `server/`.
- [x] Make sure all PR Checks are passing.
<!-- You can run the tests manually:
In `python-sdk/`, run the run `pip install -e .`, start the server and executor, `python test_graph_behaviours.py`.

### CODE REVIEW

File: python-sdk/tests/test_graph_behaviours.py [hunk 1]
```diff
@@ -208,6 +215,18 @@ def test_simple_function(self, is_remote):
         output = graph.output(invocation_id, "simple_function")
         self.assertEqual(output, [MyObject(x="ab")])
 
+    # @parameterized.expand([(False), (True)])
```
Comment (diptanu) [inline python-sdk/tests/test_graph_behaviours.py:218]: This probably shouldn't be commented.

File: python-sdk/tests/test_graph_behaviours.py [hunk 2]
```diff
@@ -208,6 +215,18 @@ def test_simple_function(self, is_remote):
...
+    def test_simple_function_multiple_inputs(self, is_remote=True):
+        print(sys.version_info.major, sys.version_info.minor)
```
Comment (diptanu) [inline python-sdk/tests/test_graph_behaviours.py:220]: Unintented?

File: python-sdk/tests/test_graph_behaviours.py [hunk 3]
```diff
@@ -28,6 +29,12 @@ def simple_function(x: MyObject) -> MyObject:
     return MyObject(x=x.x + "b")
 
 
+@indexify_function()
```
Comment (diptanu) [inline python-sdk/tests/test_graph_behaviours.py:32]: Probably worth testing if this works with json encoding as well. May be it works.

**Review by diptanu — COMMENTED** [review]
Left some comments, merge it once the tests are green!

File: python-sdk/tests/test_graph_behaviours.py (hunk 1 above)
Comment (stangirala) [reply python-sdk/tests/test_graph_behaviours.py:218]: Ran into this on my laptop testing,`ModuleNotFoundError: No module named 'test_graph_behaviours'`. I just pushed the PR to run the test and see if it worked. Will fix.

**Pull request merged**